components/DetailTabs/DetailTabs.vue

```diff
@@ -113,10 +113,34 @@ export default {
 @media screen and (max-width: 768px) {
   .details-tabs {
     &__container {
-      margin: 0;
       margin-top: 1.5rem;
+      padding-right: 1rem;
+      padding-left: 1rem;
       &--types {
-        display: -webkit-inline-box;
+        display: -webkit-box;
+        width: max-content;
+      }
+      &--button {
+        font-size: 1em;
+        padding-right: 0;
+        padding-left: 0;
+      }
+      li {
+        margin-right: 0;
+      }
+    }
+  }
+
+  ::v-deep .el-row--flex {
+    overflow-y: scroll;
+  }
+}
+
+@media screen and (max-width: 1023px) {
```

Comment:
lol, assuming this is similar to what we were chatting about yesterday - you want to target everything _up to_ but not including 1024 (Ipad Pro or original Ipad landscape), correct?

Just making sure its not a typo.

Reply:
Yessir! The goal is to have that gray line underneath the tabs section always extend to the end of the container starting from mobile all the way up to and including Ipad viewports. If 1024 includes Ipad Pro/Landscape, I can change to reflect that - just one px difference :)